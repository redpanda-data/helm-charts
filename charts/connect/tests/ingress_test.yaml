suite: ingress tests
templates:
  - ingress.yaml
tests:
  - it: should enable ingress default
    set:
      ingress:
        enabled: true
        annotations:
          test: test
        hosts:
          - host: test.local
            paths:
              - path: /
        tls:
          - secretName: test-tls
            hosts:
              - test.local
    asserts:
      - matchSnapshot:
          path: spec
