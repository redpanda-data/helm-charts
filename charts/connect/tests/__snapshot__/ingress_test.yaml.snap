should enable ingress default:
  1: |
    rules:
      - host: test.local
        http:
          paths:
            - backend:
                service:
                  name: RELEASE-NAME-redpanda-connect
                  port:
                    number: 80
              path: /
    tls:
      - hosts:
          - test.local
        secretName: test-tls
